typescript/processors/readTypeScriptModules.js

```diff
@@ -0,0 +1,451 @@
+var glob = require('glob');
+var path = require('canonical-path');
+var _ = require('lodash');
+var ts = require('typescript');
+
+module.exports = function readTypeScriptModules(tsParser, modules, getFileInfo, untouchedNamespaces,
+                                                getExportDocType, getContent, createDocMessage, log) {
+
```

Comment:
@pvadam with 5135649e87c81d887635c85a298188a9f6868c1b your example should (almost) work. I don't know how to implement the basePath though ... at the moment it would be ignored.